README.md

```diff
@@ -35,13 +50,15 @@ This can be installed via pip:
 ```
 pip install ".[plot]"
 ```
-
-
+The last step is to install the pre-commit hooks.
```

Comment:
Per the mantra "Explicit better than Implicit," could we leave it in the README but include an IF DEVELOPER clarifier before the block ... as shown below, for example. 

If you are a developer, use: pip install ".[plot]"
If you are any other type of user, then install via: "pip install plot"

Reply:
Sorry, I should have been more clear on this comment. I was referring to removing the reference to the `pre-commit install` line since we include it [here](https://github.com/NREL/mappymatch/blob/a3dd997204d0327c8d37fd6d71b3f2c5416101c2/CONTRIBUTING.md?plain=1#L96)